controllers/apiserver.go

```diff
@@ -31,7 +34,10 @@ var apiServerTemplates = []string{
 	"apiserver/deployment.yaml.tmpl",
 }
 
-func (r *DSPAReconciler) ReconcileAPIServer(dsp *dspav1alpha1.DataSciencePipelinesApplication, params *DSPAParams) error {
+// serverRoute is a Conditional resource
```

Comment:
what does `Conditional` (capital-C) resource mean?

Reply:
it just means it's a resource that is only deployed conditionally, and not always, hence why it's not in the array right above it, which are always deployed, I've updated the comments to be more clear here

Reply:
thanks